readme.md

```diff
@@ -457,6 +457,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [ChatOps](https://github.com/exAspArk/awesome-chatops) - Managing technical and business operations through a chat.
 - [Falsehood](https://github.com/kdeldycke/awesome-falsehood) - Falsehoods programmers believe in.
 - [Domain-Driven Design](https://github.com/heynickc/awesome-ddd) - Software development approach for complex needs by connecting the implementation to an evolving model.
+- [Alfred Workflows](https://github.com/derimagia/awesome-alfred-workflows)
```

Comment:
Updated, should this be in the default pull request template? I saw it but a lot of them that should have it didn't so I wasn't sure. Thanks for the review!